### ISSUE TITLE
Building Seismic Rating Data

### ISSUE THREAD

**Issue by ungoldman** [CityofPortland/pdxdata#4]
> Is there any #opendata around buildings' seismic ratings in PDX? Seems like a fun project to map (if that doesn't already exist).

\- [tweet](https://twitter.com/noahmanger/status/634021800825909248) from @noahmanger of [18f](https://18f.gsa.gov/)

@ksmpdx do you know if this data already exists? Wasn't able to find anything on http://civicapps.org and the search on http://portlandmaps.com/metadata/index.cfm seems a bit broken (no submit button, entering a search term and hitting enter doesn't seem to do the trick either).

**Comment by noahmanger** [comment]
FYI I found this https://www.portlandoregon.gov/bds/article/93088 . Not a full answer, but something.

**Comment by ksmpdx** [comment]
Yes, nothing on seismic ratings. But we do have an (imperfect) data for unreinforced masonry (URM) buildings, derived from the original inventory mentioned above. Here are some shapefiles (derived from the buildings data you can download from CivicApps):

https://dl.dropboxusercontent.com/u/2836837/building_footprints_pdx_urms_150819.zip

Put the metadata link in there as well. The STRUC_TYPE field flags URM buildings.

Note that we are in the process of updating this inventory. We'll have a much more accurate dataset in the next 3-6 months.Hope that helps!

Regarding the metadata site, search gives suggested datasets as you type. Good catch, though, that if there are no suggestions, nothing happens. No keyword search. I'll pass that along.
